build/Tests/Tests.Shared/Tests.Shared.Logger.vcxproj

```diff
@@ -135,13 +80,6 @@
       <OptimizeReferences>true</OptimizeReferences>
       <AdditionalLibraryDirectories>..\..\..\deps\prebuilt\TAEF\build\Library\$(PlatformTarget);%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
     </Link>
-    <ClangCompile>
-      <IncludePaths>..\..\..\Frameworks\include;..\..\..\Frameworks\UIKit.Xaml;..\..\..\include\xplat;..\..\..\tests\frameworks\include;..\..\..\tests\Tests.Shared;..\..\..\tests\frameworks\gtest;..\..\..\tests\frameworks\gtest\include;..\..\..\deps\prebuilt\TAEF\build\Include;..\..\..\samples\XAMLCatalog;%(IncludePaths)</IncludePaths>
-      <CompileAs>CompileAsObjCpp</CompileAs>
-      <OtherCPlusPlusFlags>-fmsvc-real-char -Wdeprecated-declarations</OtherCPlusPlusFlags>
```

Comment:
This lib is straight c++ so clang options aren't useful ... 

---
In reply to: [102294319](https://github.com/Microsoft/WinObjC/pull/1971#discussion_r102294319) [](ancestors = 102294319,102293959)